libs/platform/experience/src/feature/latest.ts

```diff
@@ -0,0 +1,36 @@
+import { AppFeature } from '@spryker-oryx/core';
+import { provideLitRoutes } from '@spryker-oryx/router/lit';
+import * as components from '../components';
+import { PreviewPlugin } from '../plugins';
+import { defaultExperienceLatestRoutes } from '../routes';
+import {
+  experiencePreviewProviders,
+  experienceProviders,
+  layoutProviders,
+} from '../services';
+
+export const experienceLatestComponents = [components.compositionComponent];
+
+export const layoutLatestFeature: AppFeature = {
```

Comment:
if there is no difference, I'd go with simple
`layoutLatestFeature = layoutFeature` for such cases. Only if we will add something new, we can provide more customized sets.